spec/features/poros/food_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+
+  describe "food class exists with atributes" do
+
+    it "has atributes" do
+
+      info = {description: "orang tea",
+              brand: "bigalo",
+              gtinUpc: "bla bla blaa",
+              ingredients: "oranges and tea leaves"
+            }
```

Comment:
When creating a poro, you'll likely be sending in a large json object, which your poro will be responsible for grabbing what it needs and creating objects out of that. So for this test, I'd suggest using fixture data again, or at least grabbing a json object for a particular food's info, and putt that in your test.